documentation/ReleaseNotes.md

```diff
@@ -0,0 +1,7 @@
+Today we are releasing the next official preview of the `dotnet monitor` tool. This release includes:
+- **Default Process**: Allow configuration of the default process for metrics and for routes where the target process is not specified. In prior releases, the default process would resolve only if there was a single observerable process. See the [configuration](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/configuration.md#default-process-configuration) documentation for more details.
```

Comment:
Did you want to comment on the fact this is a breaking change in some circumstances? I.e. Aks containers with shared process namespace may not work anymore.